app/views/users/index.html.erb

```diff
@@ -7,16 +6,25 @@
     <th>Email Address</th>
     <th>Phone Number</th>
     <th>Join Date</th>
+    <th>Status</th>
+    <th>Change Status</th>
```

Comment:
I guess we don't need a column name here we can just go with
`<th colspan="1"></th>`

Same as how rails default generator generates table, actions don't have column name